react/features/base/settings/reducer.js

```diff
@@ -53,7 +53,9 @@ const filterSubtree = {};
 
 // start with the default state
 Object.keys(DEFAULT_STATE).forEach(key => {
-    filterSubtree[key] = true;
+    if (typeof filterSubtree[key] === 'undefined') {
```

Comment:
doesn't this break something for the other keys that were initialized with `false`?

Reply:
Actually this piece of code was not doing what it was promising. There was no way of setting a default value to false (or true for that matter). This does not change the fact that these settings will be overwritten by the local storage values.

Reply:
Right, this will mess the default state.

Reply:
Fixed